onnx/test/shape_inference_test.py

```diff
@@ -2792,6 +2792,22 @@ def test_einsum_incorrect_num_inputs(self):  # type: () -> None
             [])
         self.assertRaises(checker.ValidationError, self._inferred, graph)
 
+    def test_cross_entropy_none(self):  # type: () -> None
+        graph = self._make_graph(
+            [("x", TensorProto.FLOAT, (2, 3)),
+             ("y", TensorProto.FLOAT, (2, 3))],
+            [make_node('SoftmaxCrossEntropy', ['x', 'y'], ['z'], reduction='none')],
+            [],)
+        self._assert_inferred(graph, [make_tensor_value_info('z', TensorProto.FLOAT, (2, 3))])  # type: ignore
+
+    def test_cross_entropy_mean(self):  # type: () -> None
+        graph = self._make_graph(
+            [("x", TensorProto.FLOAT, (2, 3)),
+             ("y", TensorProto.FLOAT, (2, 3))],
```

Comment:
Please add a case with the present of `weight`, if we decide not to move `weight` to the attribute list.